fedimint-cli/src/client.rs

```diff
@@ -415,7 +415,7 @@ pub async fn handle_command(
                 contract_id,
                 fee,
             } = lightning_module
-                .pay_bolt11_invoice(ln_gateway, bolt11, ())
+                .pay_bolt11_invoice(ln_gateway, bolt11, amount, ())
```

Comment:
This behavior should be documented somewhere. Ideally the `clap` help and maybe also a comment here that for BOLT11 we allow split payments.